modules/installation/pages/install.adoc

```diff
@@ -16,21 +16,21 @@ video::sfe3Q37oWJQ[youtube,800,600]
 
 Before you can install the TigerGraph system, you need the following:
 
-* One or more machines that meet the minimum xref:hw-and-sw-requirements.adoc[_Hardware and Software Requirements_].
-* A sudo user with the same username and login credential on every machine.
+* One or more machines that meet the minimum xref:hw-and-sw-requirements.adoc[_Hardware and Software Requirements_]. If your machine doesn't have the software packages necessary, the downloadable TigerGraph package includes a setup script called `install_tools.sh`.
```

Comment:
Semantically not quite right.  The downloadable TG package includes `install_tools.sh` whether or not your machine meets the requirements.  I would change the wording so that we directly tell the user to run `install_tools.sh`, unless they have already confirmed that they meet all the HW and SW requirements.